### ISSUE TITLE
Gettin AttributeError on admin from latest master version

### ISSUE THREAD

**Issue by spapas** [healthchecks/healthchecks#1050]
Hey friends, I updated to the latest version from master and getting:

```AttributeError: Unable to lookup 'channel__kind' on Notification or NotificationsAdmin```

The commit is `d3ae4e7fac6b99a6211559f79740076fe9bfe08d`.

TIA

**Comment by cuu508** [comment]
Did you upgrade packages to versions listed in requirements.txt? In particular, are you running Django 5.1?

**Comment by spapas** [comment]
Hello @cuu508 yes you are right I didn't have installed the requirements. It works now however I get another warning:

When running migrate I get:

```
 Your models in app(s): 'api' have changes that are not yet reflected in a migration, and so won't be applied. 
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

Trying makemigrations dry run I get:

python manage.py makemigrations --dry-run 

Migrations for 'api':
  hc/api/migrations/0111_alter_channel_kind.py
    ~ Alter field kind on channel      

Shouldn't that migration be alread commited  on the master ?

**Comment by cuu508** [comment]
It's related to this commit which removes a few obsolete channel kinds: https://github.com/healthchecks/healthchecks/commit/8054191be318453b6ed115648408369f40ddbbd5

You can safely ignore this warning.

**Comment by spapas** [comment]
Excellent thank you!
